app/src/main/res/layout/compose_sync_tabs_tray_layout.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.compose.ui.platform.ComposeView
```

Comment:
Wondering if we could do without having this? Mostly, recalling this from the wallpaper setting PR.